backend/server.py

```
@@ -6493,6 +6493,151 @@ def _mask(val: str | None) -> str:
+    gateway_url = f"{external_url}/openclaw"
+    pairing_token = _os.environ.get("OPENCLAW_PAIRING_TOKEN", "")
+    cli_installed = _os.environ.get("OPENCLAW_CLI_INSTALLED", "false").strip().lower() in ("true", "1", "yes")
+
+    # Check if the background gateway subprocess is alive
+    gateway_alive = False
+    try:
+        import httpx as _httpx
+        async with _httpx.AsyncClient(timeout=3.0) as client:
+            resp = await client.get(f"http://127.0.0.1:{_OPENCLAW_GATEWAY_PORT}/health")
+            gateway_alive = resp.status_code == 200
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)